meetup_v11.html

```diff
@@ -75,7 +75,7 @@ <h5 class="card-title">⭐️ Join via Zoom app</h5>
             <h5 class="card-title">Join via Zoom (browser)</h5>
             <p class="card-text">If you can't install the Zoom app, you can join the meetup using the Zoom web client.
               You can ask questions in the chat. However, you will not be able to see all attendees at once.</p>
-            <a href="#" class="btn btn-secondary disabled">Join via web client (soon)</a>
+            <a href="https://us02web.zoom.us/wc/88914554855/start" class="btn btn-secondary">Join via web client</a>
```

Comment:
We need to provide the password here since it's not encoded in the URL as above

Reply:
You are right, I just copied that link from the dashboard. Seems to be the wrong one.

In v9 we used this link for the browser version:
`https://zoom.us/wc/join/750741220?pwd=`

So I would expect that such a link should to the trick, does it?
`https://zoom.us/wc/join/88914554855?pwd=VzhpQXFzQXBkek15TXBPSHRaeVZJZz09`